doc/dune-files.rst

```diff
@@ -1691,8 +1691,11 @@ will be deprecated in Coq 8.12.
 mdx (since 2.4)
 ---------------
 
-MDX is a tool that helps you keep your markdown documentation up to date by
+MDX checks if the markdown files in the directory of your MDX stanza have changed. 
+It can help you keep your markdown documentation up to date by
 checking that the code examples it contains are correct.
+It can also be used as a testing tool: the checks carried out by MDX are 
```

Comment:
I'd rephrase that a bit since it's not really a choice and therefore the `It can` is a bit misleading. How about something along the lines of:
```
When setting an MDX stanza, the checks carried out by MDX are automatically attached to the ``runtest`` alias of the stanza's directory.
```